contracts/registry/VersionRegistry.sol

```diff
@@ -5,6 +5,8 @@ pragma solidity ^0.4.24;
 /// @dev keeps track of the latest version of all contracts
 contract VersionRegistry {
 
+  event VersionSet(string contractName, address proxyAddress, string versionName, address newImplementation);
```

Comment:
Can strings be indexed? If so, I think it would be useful to index on `contractName`. It would also be useful to index on `proxyAddress`